integration-tests/contracts/OVMMulticall.sol

```diff
@@ -45,7 +49,9 @@ contract OVMMulticall {
     }
 
     function getCurrentBlockNumber() public view returns (uint256 blockNumber) {
-        blockNumber = block.number;
+        blockNumber = iOVM_L1BlockNumber(
```

Comment:
Why did you update this but not inside of `aggregate`? I feel like the intended behavior of multicall would be "return the L2 blocknumber"